GVRf/Framework/framework/src/main/java/org/gearvrf/GVRRenderTexture.java

```diff
@@ -53,10 +53,18 @@ public GVRRenderTexture(GVRContext gvrContext, int width, int height, int sample
         this(gvrContext,width,height,sampleCount,1);
     }
 
+    public GVRRenderTexture(GVRContext gvrContext, int width, int height, int sampleCount, boolean monoscopic)
```

Comment:
Can't push backend specifics into the framework. In a future change the monoscopic backend will become its own separate backend.

Please create a more abstract way of requesting and producing on and off screen targets.